config/locales/de.yml

```diff
@@ -279,25 +279,25 @@ de:
-      rule3: "No guests are allowed. Our workshops are already overcrowded and we cannot make any exceptions. If your friend meets our eligibility criteria, get them to sign up and RSVP themselves."
-      rule4: "Please be respectful of our hosts. Do not show up exceedingly early to a workshop (not earlier than 15 minutes before the event begins) and try to leave the workstation clean and as you found it."
-      rule5: "<u>Please do not work on projects that were assigned as part of any bootcamp or university curriculum.</u> Talk to your course adviser if you feel like you need additional support with your coursework. We welcome your attendance, but we encourage you to use your time at codebar to develop skills that are outside the normal scope of your studies."
-      rule6: "Do not bring along and ask for help with any commercial work. This involves work you're being paid to do."
-      rule7: "Projects that involve specific frameworks/libraries (Rails, NodeJS, Meteor, Angular, Ember, React, etc) or languages that are not part of the codebar tutorials (anything other than Ruby or Javascript) are welcome but we cannot guarantee that a coach will be available to help you. In these cases we recommend selecting a codebar tutorial or katas (codewars.com or exercism.io) as a back-up option."
+      intro: "codebar ist und war schon immer offen für alle Arten von Programmiererfahrung. Die meisten Tutorials auf unserer Website richten sich zwar an Einsteiger in die Webentwicklung, aber wir wollen unsere Politik der offenen Tür für alle Studenten fortsetzen, die bei ihren eigenen Projekten Unterstützung durch einen Coach benötigen. Da unsere Workshops häufig überzeichnet sind, bitten wir die Studenten, die folgenden Erwartungen an die Teilnahme an unseren Workshops zu erfüllen"
+      rule1: "Um an einem Workshop teilnehmen zu können, <u>müssen Sie auf der RSVP-Liste stehen.</u> Wenn Sie nicht auf die RSVP-Liste geantwortet haben oder Ihnen kein Platz zugewiesen wurde, werden Sie nicht zu einem Workshop zugelassen."
+      rule2: "Wenn Sie an einem Workshop nicht mehr teilnehmen können, <u>sagen Sie bitte Ihre RSVP bis spätestens 12:00 Uhr </u> am Tag des Workshops ab. Wir haben Verständnis dafür, dass Notfälle in letzter Minute auftreten - in diesem Fall setzen Sie sich bitte so schnell wie möglich direkt mit einem Veranstalter in Verbindung. Wiederholtes Nichterscheinen führt zu einer vorübergehenden oder dauerhaften Aussetzung Ihrer Einladungen."
+      rule3: "Es sind keine Gäste zugelassen. Unsere Workshops sind bereits überfüllt und wir können keine Ausnahmen machen. Wenn Ihr Freund oder Ihre Freundin unsere Teilnahmekriterien erfüllt, sollten Sie ihn/sie dazu bringen, sich selbst anzumelden und zu bestätigen."
```

Comment:
`Gäste sind nicht erlaubt`.

`Unsere Workshops sind oft sehr voll.`

---

There's another fundamental decision that applies elsewhere too to be found here:

```
Freund oder Ihre Freundin
```

It would probably make sense to use the "Genderstern" in such cases and make it `Freund*in`.